File: backend/src/main/java/cha_revelacao/dto/response/GaleriaPostResponse.java
package cha_revelacao.dto.response;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class GaleriaPostResponse {
    private Long id;
    private String mensagem;
    private String urlFoto;
    private String urlVideo;
    private LocalDateTime dataCriacao;
    private UsuarioResponse usuario;
    private EventoResponse evento;
}

File: backend/src/DTOs/Responses/GaleriaPostResponse.cs
namespace MemuVie.Evento.DTOs.Responses;

public class GaleriaPostResponse
{
    public long Id { get; set; }
    public string? Mensagem { get; set; }
    public string? UrlFoto { get; set; }
    public string? UrlVideo { get; set; }
    public DateTime DataCriacao { get; set; }
    public UsuarioResponse Usuario { get; set; } = null!;
    public EventoSummaryResponse Evento { get; set; } = null!;
}
